modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -318,6 +318,61 @@ - (void)handleSignal:(NSString *)signal
     callback(@[result]);
 }
 
+//////////////////////////////////////////////////////////////////// getConnectionStringForBootstrappingAnotherDevice
```

Comment:
is this a common formatting we are using in the codebase? please adjust to a standard format if not.

Reply:
yep this is how other functions have comments :)

Reply:
Jeez... There are not enough forward slashes here, I think we should double ;)

Reply:
I have no preference here as well, there is some mixed formatting for sure but these forward slashes seem to be present above 57 functions and absent above few of them, 
I could edit and remove all of these slashes but I don't wish to upset someone else so i dunno  :D

Reply:
imo - we remove these comments.. it seems they just comment with the name of the function. No value added there.
But definitely not in this p.r 😅

Reply:
https://github.com/status-im/status-mobile/issues/14524